javascript-if/main.js

```diff
@@ -0,0 +1,83 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+isUnderFive(4);
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+isEven(3);
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+startsWithJ('don');
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+isOldEnoughToDrink(13);
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+isOldEnoughToDrive(12);
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+isOldEnoughToDrinkAndDrive(21);
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 7) {
+    return 'acidic';
```

Comment:
`pH` levels below `0` should not be labeled `"acidic"`:

---

![image](https://user-images.githubusercontent.com/16143771/169143122-771d14f0-3fae-4a4d-a1fa-1ce3b60bce9a.png)